app/experimenter/nimbus-ui/src/components/PageEditOverview/index.tsx

```diff
@@ -65,14 +68,16 @@ const PageEditOverview: React.FunctionComponent<PageEditOverviewProps> = () => {
 
   return (
     <AppLayoutWithExperiment title="Overview" testId="PageEditOverview">
-      {({ experiment }) => {
+      {({ experiment, review }) => {
         currentExperiment.current = experiment;
+        refetchReview.current = review.refetch;
```

Comment:
I think this is the approach we'll have to take if we're going to be passing back experiment and review with `{children({ experiment, review })}`.

I want to r+ this, but am curious if @lmorchard has input here on a possible improvement because whatever approach is laid out in this PR will be used on the Audience and Branches pages as well.

Reply:
I still kinda don't like the code-smell of this, but have yet to come up with anything better. I wouldn't block on it. If I (or anyone else) come up with something, it'll be a refactor at some later point